recipes/repeatmasker/build.sh

```diff
@@ -5,10 +5,17 @@ mkdir -p ${PREFIX}/bin
 mkdir -p ${RM_DIR}
 cp -r * ${RM_DIR}
 
+export LANGUAGE=en_US.UTF-8
```

Comment:
Why is this needed?

Reply:
There were some perl warning about that in previous build, they're gone with that change